config/mclag.py

```diff
@@ -1,8 +1,12 @@
 
 import click
 from swsscommon.swsscommon import ConfigDBConnector
+from .validated_config_db_connector import ValidatedConfigDBConnector
 import ipaddress
-
+import jsonpatch
+from jsonpatch import JsonPatchConflict
```

Comment:
Can you use the same import method?
"import jsonpatch" or "from jsonpatch import xxx"

Reply:
cleaned import statement